docs/API.md

```diff
@@ -139,12 +139,12 @@ Basic data type: object
 DS for [Delegation Signer](https://tools.ietf.org/html/rfc4034) references DNSKEY-records in the sub-delegated zone.
 
 Properties:
-* `"digest"`: A string, required. Either 40 or 64 hexadecimal characters (case insensitive).
+* `"digest"`: A string, required. Either 40, 64 or 96 hexadecimal characters (case insensitive).
 * `"algorithm"`: An non negative integer, required.
 * `"digtype"`: An non negative integer, required.
 * `"keytag"`: An non negative integer, required.
 
-Extra properties in *DS info* objects are ignored when present in RPC method arguments, and never returned as part of RPC method results.
+No extra properties are allowed.
```

Comment:
I think this statement should be moved to "the top", i.e. apply for all objects.

Reply:
In the Protocol section we state that "Extra properties in the "params" object are allowed for some methods but ignored for others.". Do you know if we still have any methods that allow extra properties? I thought we got rid of that when we added the validation layer.

Reply:
> I think this statement should be moved to "the top", i.e. apply for all objects.

Done.

> Do you know if we still have any methods that allow extra properties?

None that I have seen.